packages/sql-mapper/lib/entity.js

```diff
@@ -340,7 +339,7 @@ async function buildEntity (db, sql, log, table, queries, autoTimestamp, schema,
       acc[column.column_name].enum = column.column_type.match(/'(.+?)'/g).map(enumValue => enumValue.slice(1, enumValue.length - 1))
     }
 
-    if (autoTimestamp && (column.column_name === 'updated_at' || column.column_name === 'inserted_at')) {
+    if (autoTimestamp.enabled && (column.column_name === autoTimestamp.createdAt || column.column_name === autoTimestamp.updatedAt)) {
```

Comment:
would you mind removing `enabled`? we can assume that if `options.autotimestamp` is false this is disabled.